virttest/test_setup.py

```diff
@@ -1887,19 +1888,29 @@ def __init__(self, params):
         if distro_obj.name.lower().strip() == 'ubuntu':
             self.polkit_name = "polkitd"
         self.polkitd = service.Factory.create_service(self.polkit_name)
-
-        if self.params.get("action_id"):
-            self.action_id = self.params.get("action_id").split()
-        else:
-            self.action_id = []
         self.user = self.params.get("unprivileged_user")
-        if self.params.get("action_lookup"):
-            # The action_lookup string should be separated by space and
-            # each separated string should have ':' which represent key:value
-            # for later use.
-            self.attr = self.params.get("action_lookup").split()
+        # For libvirt-5.6.0 and later, use 'action_matrix' for all rules
+        # required to setup polkit.
+        # For libvirt before 5.6.0, still use 'action_id' and 'action_lookup'
+        # for all rules required to setup polkit.
+        self.action_list = []
+        if (libvirt_version.version_compare(5, 6, 0) and
```

Comment:
where should it go if version > 5.6.0 and no "action_matrix" given?

Reply:
@kylazhang , this is to be compatible to existing cases which have not provided 'action_matrix' parameter. If version > 5.6.0 and no "action_matrix" given， 'action_id' and 'action_lookup' still are used as before.  The case will pass if it does not require the new rules, or fail if it does.

Reply:
ok, thanks